pkg/container/resolver_test.go

```diff
@@ -104,8 +111,88 @@ func TestResolverFail(t *testing.T) {
 		common.ToPtr(""),
 		common.ToPtr(false),
 		false,
+		nil,
 	})
 	specs, err = resolver.Finish()
 	assert.Error(t, err)
 	assert.Len(t, specs, 0)
 }
+
+func TestResolverLocalManifest(t *testing.T) {
+	currentUser, err := user.Current()
+	if err != nil {
+		t.Fatalf("Failed to get current user: %v", err)
+	}
+
+	if currentUser.Uid != "0" {
+		t.Skip("User is not root, skipping test")
+	}
+
+	_, err = exec.LookPath("podman")
+	if err != nil {
+		t.Skip("Podman not available, skipping test")
+	}
+
+	containerFile, err := os.CreateTemp("", "Containerfile")
```

Comment:
(nitpick) there is `t.TempDir()`  in go now so `os.CreateTemp(t.TempDir(), "Containerfile")` could be used to avoid avoid to do the defer os.Remove as golang will auto-cleanup the dir